packages/zoe/src/contracts/loan/types.js

```diff
@@ -76,12 +76,31 @@
  *   when liquidation is triggered
  */
 
+/**
+ * @typedef BorrowerConfigPropertiesMinusDebt
+ *
+ * @property {ZCFSeat} collateralSeat
+ *
+ *   The ZCFSeat holding the collateral in escrow after the borrower
+ *   escrows it
+ *
+ * @property {PromiseKit} liquidationPromiseKit
+ *
+ *   PromiseKit that includes a promise that resolves to a PriceQuote
+ *   when liquidation is triggered
+ */
+
 /**
  * @typedef {LoanConfigWithLender & BorrowerConfigProperties } LoanConfigWithBorrower
  *
  * The loan has a lender, a borrower, and collateral escrowed.
  */
 
+/**
+ * @typedef {LoanConfigWithLender & BorrowerConfigPropertiesMinusDebt
+ * } LoanConfigWithBorrowerMinusDebt
```

Comment:
It's unusual to see typedefs broken with a curly starting the line. When it's less than ~100 characters, I prefer the declared name at the end of the line.  

Your choice here, as I don't think we have standard formatting rules

Reply:
Ah, this was an automatic change from my linters. Let me see if I can undo it.

Reply:
Ah, actually, I just redid this, and this particular formatting was already there. I didn't change it in this PR, so I'm going to punt any fixing to a different PR if desired